File: webSISGERT/Models/Cotizaciones/Cotizacion_OT.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using webSISGERT.Models.OT;

namespace webSISGERT.Models.Cotizaciones
{
    public class Cotizacion_OT
    {
        public int CotizacionId { get; set; }
        public int OrdenTrabajoId { get; set; }
        public virtual Cotizacion cotizacion { get; set; }
        public virtual OrdenTrabajo ordenTrabajo { get; set; }
    }
}

File: webSISGERT/ClientApp/src/app/cotizaciones/Cotizacion_OT.ts
import { iCotizacion } from "./cotizacion";
import { iOrdenTrabajo } from "../ordenes-trabajo/ordenTrabajo";

export interface iCotizacion_OT {
  cotizacionId: number,
  ordenTrabajoId: number,
  cotizacion: iCotizacion,
  ordenTrabajo: iOrdenTrabajo
}
